cypress/integration/core-opensearch-dashboards/opensearch-dashboards/apps/data_explorer/date_nanos.spec.js

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {
+  TestFixtureHandler,
+  MiscUtils,
+} from '@opensearch-dashboards-test/opensearch-dashboards-test-library';
+const testFixtureHandler = new TestFixtureHandler(
+  cy,
+  Cypress.env('openSearchUrl')
+);
+
+const miscUtils = new MiscUtils(cy);
+
+describe('date_nanos', () => {
+  const fromTime = 'Sep 21, 2019 @ 20:31:44.000';
+  const toTime = 'Sep 23, 2019 @ 03:31:44.000';
+  before(() => {
+    // import date nanos
+    testFixtureHandler.importJSONMapping(
+      'cypress/fixtures/dashboard/opensearch_dashboards/data_explorer/date_nanos/mappings.json.txt'
+    );
+
+    testFixtureHandler.importJSONDoc(
+      'cypress/fixtures/dashboard/opensearch_dashboards/data_explorer/date_nanos/data.json.txt'
+    );
+    cy.setAdvancedSetting({
+      defaultIndex: 'date-nanos',
+    });
+
+    miscUtils.visitPage('app/data-explorer/discover#/');
```

Comment:
do you think it's low hanging fruit enough to make this configurable to where I can pass a cypress env variable if I want legacy discover or current discover?

Reply:
LEgacy discover will be removed in the next release, so it does not make sense to switch between the two